core-rust/state-manager/src/store/rocks_db.rs

```diff
@@ -97,114 +98,128 @@ use crate::transaction::{
-    TxnByStateVersion,
-    TxnIdentifiersByStateVersion,
-    LedgerReceiptByStateVersion,
-    LocalTransactionExecutionByStateVersion,
-    StateVersionByTxnIntentHash,
-    StateVersionByTxnUserPayloadHash,
-    StateVersionByTxnLedgerPayloadHash,
-    LedgerProofByStateVersion,
-    LedgerProofByEpoch,
+    StateVersionToRawLedgerTransactionBytes,
+    StateVersionToCommittedTransactionIdentifiers,
+    StateVersionToLedgerTransactionReceipt,
+    StateVersionToLocalTransactionExecution,
+    IntentHashToStateVersion,
+    NotarizedTransactionHashToStateVersion,
+    LedgerTransactionHashToStateVersion,
+    StateVersionToLedgerProof,
+    EpochToLedgerProof,
     Substates,
-    SubstateNodeAncestryRecords,
+    NodeIdToSubstateNodeAncestryRecord,
     VertexStore,
-    StateHashTreeNodes,
-    StaleStateHashTreeNodeKeysByStateVersion,
-    TransactionAccuTreeSliceByStateVersion,
-    ReceiptAccuTreeSliceByStateVersion,
-    ExtensionsData,
+    NodeKeyToTreeNode,
+    StateVersionToStaleTreeParts,
+    StateVersionToTransactionAccuTreeSlice,
+    StateVersionToReceiptAccuTreeSlice,
+    ExtensionsDataKeyToCustomValue,
     AccountChangeStateVersions,
-    ExecutedGenesisScenarios,
+    ScenarioSequenceNumberToExecutedGenesisScenario,
 ];
 
 impl fmt::Display for RocksDBColumnFamily {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        let str = match self {
-            TxnByStateVersion => "txn_by_state_version",
-            TxnIdentifiersByStateVersion => "txn_identifiers_by_state_version",
-            LedgerReceiptByStateVersion => "ledger_receipt_by_state_version",
-            LocalTransactionExecutionByStateVersion => {
-                "local_transaction_execution_by_state_version"
-            }
-            StateVersionByTxnIntentHash => "state_version_by_txn_intent_hash",
-            StateVersionByTxnUserPayloadHash => "state_version_by_txn_user_payload_hash",
-            StateVersionByTxnLedgerPayloadHash => "state_version_by_txn_ledger_payload_hash",
-            LedgerProofByStateVersion => "ledger_proof_by_state_version",
-            LedgerProofByEpoch => "ledger_proof_by_epoch",
-            Substates => "substates",
-            SubstateNodeAncestryRecords => "substate_node_ancestry_records",
-            VertexStore => "vertex_store",
-            StateHashTreeNodes => "state_hash_tree_nodes",
-            StaleStateHashTreeNodeKeysByStateVersion => "stale_state_hash_tree_node_keys",
-            TransactionAccuTreeSliceByStateVersion => {
-                "transaction_accu_tree_slice_by_state_version"
-            }
-            ReceiptAccuTreeSliceByStateVersion => "receipt_accu_tree_slice_by_state_version",
-            ExtensionsData => "extensions_data",
-            AccountChangeStateVersions => "account_change_state_versions",
-            ExecutedGenesisScenarios => "executed_genesis_scenarios",
-        };
-        write!(f, "{str}")
+        f.write_str(self.into())
```

Comment:
_(note from author)_

If there is some legitimate reason why it was done the manual way before - please shout!

Reply:
I think I understand... any change here effectively requires a ledger-wipe :(

Reply:
As discussed with @malexandru-rdx on Slack:
It's important to not make it refactoring-sensitive. Especially with people on the team that love refactorings. Hence I bring back the manual mappings here.

Reply:
(I also changed some names to be more like "naming a concept" and not "describing the schema", since these should be as future proof as possible)